pkgs/top-level/php-packages.nix

```diff
@@ -142,6 +142,17 @@ let
     ];
   };
 
+  event = buildPecl rec {
+    version = "2.5.0";
+    pname = "event";
+
+    sha256 = "1igbxla4s784z7lw1jar6pjyfn596040a52kfmawwclqf9qcvx0v";
+
+    configureFlags = [ "--with-event-libevent-dir=${libevent.dev}" ];
+    nativeBuildInputs = [ pkgs.pkgconfig ];
+    buildInputs = [ openssl libevent ];
```

Comment:
`buildInputs = with pkgs; [ openssl libevent ];`

This would remove the need of adding openssl and libevent as arguments to the php-packages.nix file.